labs/b8.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Lab 8 - Security Fundamentals
+layout: lab
+---
+
+For the purpose of this lab, we will be primarily using [GnuPG](https://gnupg.org) (also referred to as GPG), a free implementation of the [OpenPGP standard](https://openpgp.org/). As stated by GPG's website:
+>GnuPG allows you to encrypt and sign your data and communications; it features a versatile key management system, along with access modules for all kinds of public key directories.
+
+## Encryption and Decryption
+Encryption takes in input and a key, returning garbled output. Decryption takes in a key and the garbled output from encryption, recovering and returning the original input only if the decryption key is valid.
```

Comment:
I use plaintext/ciphertext in my slides; i'll just link it at the beginning. I don't know if I want my lab to end up being an entire lecture (instead of a refresher) when people just wanna do the lab.